Maquetado/scss/tech.scss

```diff
@@ -0,0 +1,51 @@
+$color: #b2db59;
+
+.tech {
+  align-items: center;
+  background-color: $color;
+  border-radius: 10px;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
```

Comment:
Spacing between elements can be handled using margin bottom in every element and using justify content flex start. By doing space around a longer text in the description means less space beteen elements.